### DISCUSSION TITLE
求助导入mc包

### DISCUSSION THREAD

**Discussion by zdsy03** [share121/inter-knot#5348] [委托]
已经安装了mc，为什么.mcpack文件的打开方式没有mc呀，所有的教程都是有mc选项，我以前也是这么导入的，是因为mc更新了不能导入了吗
![Image_1727114148645](https://github.com/user-attachments/assets/30ac5d2a-6183-4862-b738-493307fd638b)，
![Image_1727114148652](https://github.com/user-attachments/assets/61e2ab2a-3ab8-446b-bcc2-e6e0a0929b40)

**Comment by elysiam2** [comment]
建议使用mt管理器

**Comment by ghost** [comment]
下个es文件浏览器，找到你那个文件，单击一下，它会弹出打开方式。
![Screenshot_2024-09-25-12-05-50-263_com estrongs android pop](https://github.com/user-attachments/assets/971ed378-6cb8-44b9-9690-3ed3268dc0c7)
